packages/react-hooks/src/useBalance.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Balance } from '@polkadot/types/interfaces';
+
+import { DeriveBalancesAll } from '@polkadot/api-derive/types';
+import { useApi, useCall } from '@polkadot/react-hooks';
+
+/**
+ * Gets the account balance available
+ *
+ * @param accountId - TODO: what does 'null' mean? a default account id for some context?
+ */
+export function useBalance (accountId: string | null): Balance | undefined {
```

Comment:
I believe the `useBalance` hook is not used anywhere and can be removed.

Reply:
I planned to consume it in Bounties (atm it's duplicated), but atm it's only a single use in Bounties. So I'll follow you suggestion.